### PR TITLE
Use test_passed instead of test_failed

### PR DESCRIPTION
**Pull request by lvdheever**

### CODE REVIEW

File: nosekatreport/plugin.py [hunk 1]
```diff
@@ -293,8 +294,8 @@ def set_step_state(self, state, message=None):
                 # Do not add the traceback for skipped steps
                 stack = traceback.format_stack()
                 action['stack'] = stack
-                Aqf.log_traceback('Last 6 lines of stack:\n' +
-                                  ' '.join(stack[-7:-1]))
+                Aqf.log_traceback('Last few lines of stack:\n' +
```
Comment (mmphego) [inline nosekatreport/plugin.py:297]: Only printing the important bits and pieces to stdout, excluding system logs

**Review by mmphego — APPROVED** [review]
CBF Approves changes from test passed instead of test failed

**Comment by mmphego** [comment]
Test on your side, and we can merge to master.

**Comment by lvdheever** [comment]
CAM tests are working well enough for this to be merged. There are some changes but we can work through these.

**Pull request merged**